README.md

```diff
@@ -116,6 +117,7 @@ Apache 2.0 license, except for contributions copied from Vim (identified by the
 
 [license-commit]: https://github.com/neovim/neovim/commit/b17d9691a24099c9210289f16afb1a498a89d803
 [nvim-features]: https://neovim.io/doc/user/vim_diff.html#nvim-features
+[nvim-news]: https://neovim.io/doc/user/news.html
```

Comment:
this file can use the "flow" layout, so let's add it to this opt-in list: https://github.com/neovim/neovim/blob/06f4edc864479fb121d514ec1e7253dba85b740d/scripts/gen_help_html.lua#L42